Doc/library/importlib.resources.abc.rst

```diff
@@ -43,7 +43,7 @@
     :const:`None`. An object compatible with this ABC should only be
     returned when the specified module is a package.
 
-    .. versionadded:: 3.7
+    .. versionadded:: 3.11
 
```

Comment:
```suggestion
```

Reply:
maybe I should fix this in another PR so it can be backported to 3.11?

Reply:
Sure, or make a new PR into 3.12 and backport that 3.11?